include/seqan3/io/alignment_file/input_format_concept.hpp

```diff
@@ -157,6 +162,7 @@ SEQAN3_CONCEPT AlignmentFileInputFormat =
  * \param[out]    ref_id      The buffer for seqan3::field::REF_ID input.
  * \param[out]    ref_offset  The buffer for seqan3::field::REF_OFFSET input.
  * \param[out]    align       The buffer for seqan3::field::ALIGNMENT input.
+ * \param[out]    cigar       The buffer for seqan3::field::CIGAR input.
```

Comment:
add `cigar_type & cigar` also to the parameter list of `\fn void read` above (lines 128-133)